cpp/test/Ice/ami/AllTests.cpp

```diff
@@ -1258,26 +1258,32 @@ allTests(TestHelper* helper, bool collocated)
 
     if (p->ice_getConnection())
     {
-        cout << "testing back pressure... " << flush;
+        // Not all servers support back pressure via thread pool starvation (Swift).
+        if (!helper->isCrossTest())
```

Comment:
Doesn't this prevent to run this with all cross tests? In other places we have Slice operations to check if the server supports a feature.

But maybe is simpler to filter Ice/ami when doing Swift cross testing, using the --rfilter arg